docs/.vitepress/vitepress/components/globals/overview.vue

```diff
@@ -106,6 +108,10 @@ const getIcon = (link: string) => {
   const name = link.split('/').pop()
   return name ? overviewIcons[name] : null
 }
+
+onMounted(() => {
+  $search.value?.focus()
+})
```

Comment:
```suggestion
onMounted(() => {
  nextTick(() => {
    searchRef.value?.focus()
  })
})
```

或许还是需要在 nextTick 中执行，现在这样输入框没有聚焦样式